### PR TITLE
rename multi-master to active-active

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/mirror-main.go [hunk 1]
```diff
@@ -218,7 +223,7 @@ type mirrorJob struct {
 	excludeOptions []string
 	encKeyDB       map[string][]prefixSSEPair
 
-	multiMasterEnable bool
+	activeActiveEnable bool
```
Comment (klauspost) [inline cmd/mirror-main.go:226]:
```suggestion
	activeActive bool
```

File: cmd/mirror-main.go [hunk 2]
```diff
@@ -653,22 +658,22 @@ func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch,
...
+		disableMultipart:   disableMultipart,
+		excludeOptions:     excludeOptions,
+		olderThan:          olderThan,
+		newerThan:          newerThan,
+		storageClass:       storageClass,
+		userMetadata:       userMetadata,
+		encKeyDB:           encKeyDB,
+		statusCh:           make(chan URLs),
+		watcher:            NewWatcher(UTCNow()),
+		activeActiveEnable: activeActiveEnable,
```
Comment (klauspost) [inline cmd/mirror-main.go:676]:
```suggestion
		activeActive: activeActive,
```

File: cmd/mirror-main.go [hunk 3]
```diff
@@ -643,8 +648,8 @@ func (mj *mirrorJob) mirror(ctx context.Context, cancelMirror context.CancelFunc
 	return mj.monitorMirrorStatus()
 }
 
-func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch, isPreserve, multiMasterEnable bool, excludeOptions []string, olderThan, newerThan string, storageClass string, userMetadata map[string]string, encKeyDB map[string][]prefixSSEPair, md5, disableMultipart bool) *mirrorJob {
-	if multiMasterEnable {
+func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch, isPreserve, activeActiveEnable bool, excludeOptions []string, olderThan, newerThan string, storageClass string, userMetadata map[string]string, encKeyDB map[string][]prefixSSEPair, md5, disableMultipart bool) *mirrorJob {
+	if activeActiveEnable {
```
Comment (klauspost) [inline cmd/mirror-main.go:652]:
```suggestion
	if activeActive {
```

File: cmd/mirror-main.go [hunk 4]
```diff
@@ -643,8 +648,8 @@ func (mj *mirrorJob) mirror(ctx context.Context, cancelMirror context.CancelFunc
 	return mj.monitorMirrorStatus()
 }
 
-func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch, isPreserve, multiMasterEnable bool, excludeOptions []string, olderThan, newerThan string, storageClass string, userMetadata map[string]string, encKeyDB map[string][]prefixSSEPair, md5, disableMultipart bool) *mirrorJob {
-	if multiMasterEnable {
+func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch, isPreserve, activeActiveEnable bool, excludeOptions []string, olderThan, newerThan string, storageClass string, userMetadata map[string]string, encKeyDB map[string][]prefixSSEPair, md5, disableMultipart bool) *mirrorJob {
```
Comment (klauspost) [inline cmd/mirror-main.go:651]:
```suggestion
func newMirrorJob(srcURL, dstURL string, isFake, isRemove, isOverwrite, isWatch, isPreserve, activeActive bool, excludeOptions []string, olderThan, newerThan string, storageClass string, userMetadata map[string]string, encKeyDB map[string][]prefixSSEPair, md5, disableMultipart bool) *mirrorJob {
```

**Comment by harshavardhana** [comment]
fixed the build issue PTAL @kannappanr @vadmeste need this for making a new release.

**Pull request merged**